tests/ca-less/install_server_without_ca.yml

```diff
@@ -1,15 +1,15 @@
 ---
 - name: Generate certificates
   hosts: localhost
-  gather_facts: false
+  gather_facts: true
 
   tasks:
   - name: Run generate-certificates.sh
     command: >
       /bin/bash
       generate-certificates.sh create
-      "{{ groups.ipaserver[0] }}"
-      "{{ ipaserver_domain | default(groups.ipaserver[0].split('.')[1:] | join ('.')) }}"
+      "{{ ansible_fqdn }}"
```

Comment:
This is not correct, the task is executed on the controller. `ansible_fqdn` is not `groups.ipaserver[0]`.

Reply:
Same case in tests/external-signed-ca-with-automatic-copy/install-server-with-external-ca-with-automatic-copy.yml.
I'll look for the playbooks running in localhost and undo the changes in those.

Reply:
I've updated the PR undoing the changes for the localhost cases.